docs/commandline.rst

```diff
@@ -4,3 +4,37 @@
 
 .. autoprogram:: doctr.__main__:get_parser()
    :prog: doctr
+
+
+Configuration
+-------------
+
+In addition to command line arguments you can configure ``doctr`` using the
+``.travis.yml`` files. Command line arguments take precedence over the value
+present in the configuration file.
+
+The configuration parameters available from the ``.travis.yml`` file often
+mirror their command line siblings except dashes ``-`` are replaced by
+underscores ``_``. Some parameters are not available from configuration files
+for security reasons.
```

Comment:
> Can we be more specific here? Are there any values that aren't the same after replacing - with _? Which flags aren't available for security reasons?

Hum, I don't remember. I'm using `safe_load` from Yaml so itm might be due to that. I'll rephrase.